### PR TITLE
adopt upstream recipe test imports

### PR DESCRIPTION
**Pull request by h-vetinari**
Upstream cvxpy has a conda-recipe as well, and since (more or less) recently, this is running the test-suite as part of the build, see https://github.com/cvxgrp/cvxpy/commit/eff752f1d1151017a1cbe1a8c8a0abb12ebaa855

~I brought up the possibility of running test suite [before](https://github.com/conda-forge/cvxpy-feedstock/pull/36#issuecomment-637772911), but didn't receive an answer. Now I'm just going ahead and seeing what comes out (also slightly motivated by seeing cvxgrp/cvxpy#1003). Fingers crossed~

Nevermind, it's been running in this recipe for a while [already](https://github.com/conda-forge/cvxpy-feedstock/blame/master/recipe/meta.yaml#L48); this means this PR just syncs the import tests.

### CODE REVIEW

**Comment by h-vetinari** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -46,6 +46,7 @@ outputs:
         - nose
       commands:
         - nosetests cvxpy
```
Comment (h-vetinari) [inline recipe/meta.yaml:48]: Huh, silly me, the test suite is already run here. 🥳

**Comment by h-vetinari** [comment]
There's something curious happening with the multiprocess dependency for pypy on aarch64:

```
Package python_abi conflicts for:
cvxpy-base==1.1.1=py36h01e7d0c_0 -> python_abi==3.6[build=*_pypy36_pp73]
cvxpy-base==1.1.1=py36h01e7d0c_0 -> multiprocess -> python_abi[version='3.6.*|3.8.*|3.7.*',build='*_cp38|*_cp36m|*_cp37m']
...
```

It doesn't show a pypy-compatible build for multiprocess, even though I can see that the last [CI run](https://cloud.drone.io/conda-forge/multiprocess-feedstock/6/1/2) on the feedstock was successful:
> Uploaded /drone/src/build_artifacts/linux-aarch64/multiprocess-0.70.9-py36h97a6639_1.tar.bz2

That file also remains available in the [channel](https://anaconda.org/conda-forge/multiprocess/files). Any ideas @conda-forge/help-pypy, @conda-forge/multiprocess?

**Comment by h-vetinari** [comment]
@conda-forge/core 
Even with a successful new `multiprocess` build a couple hours ago, the aarch64+pypy combination cannot find the required package (or at least, I'm not able to decipher the conflict log). What can I do to unblock this?

**Comment by isuruf** [comment]
Have you tried creating the same environment locally?

**Comment by h-vetinari** [comment]
> Have you tried creating the same environment locally?

I don't have an aarch64 machine available, but the weird thing is that this used to build fine until recently, otherwise the last PR (#37) wouldn't have gone through.

**Comment by isuruf** [comment]
Do you have docker? Then you can use the condaforge image which has qemu installed.

**Comment by h-vetinari** [comment]
> Do you have docker? Then you can use the condaforge image which has qemu installed.

Which condaforge image? `condaforge/linux-anvil-aarch64:latest?`. I might be able to try tomorrow, but I don't expect a different result. Have you had a look at the conflict logs - anything else suspicious you see that I'm overlooking?

**Comment by isuruf** [comment]
> Which condaforge image? condaforge/linux-anvil-aarch64:latest?

Yes.

> I might be able to try tomorrow, but I don't expect a different result.

Try installing packages one by one.

**Comment by h-vetinari** [comment]
Rebased on the passing 1.1.5 - hope this is fine now.

**Comment by h-vetinari** [comment]
@isuruf 

After building correctly on aarch+pypy (which had been a problem in this PR [before](https://github.com/conda-forge/cvxpy-feedstock/pull/38#issuecomment-644417713) for some reason), it's back to dependency resolution failure. The one information gain is that it's not due to `multiprocess` because that wasn't used in 7a9e597.

**Comment by h-vetinari** [comment]
Hm, the actual error now seems to be:
```
ld: cannot find /usr/lib64/libpthread_nonshared.a
```

**Comment by scopatz** [comment]
Yeah, we got rid of static libraries, it is probably best to change that to link to the shared lib, `libpthread_nonshared.so`

**Comment by h-vetinari** [comment]
Thanks @scopatz. Is the shared library really called `_nonshared`?! Also, where in the recipe is this supposed to appear, resp. be changed? I cannot find the string "shared"/"nonshared" anywhere in the git repo for this feedstock.

**Comment by isuruf** [comment]
> Hm, the actual error now seems to be:

> ld: cannot find /usr/lib64/libpthread_nonshared.a

Can you post a log with this error?

**Comment by h-vetinari** [comment]
@conda-forge-admin, please rerender

**Comment by h-vetinari** [comment]
This fortunately resolved itself (build ran through after re-render, also after merge) - thanks for jumping in to help @scopatz, @isuruf!

**Pull request merged**